### ISSUE TITLE
Forbid the "Select all" option

### ISSUE THREAD

**Issue by kprkpr** [roundcube/roundcubemail#7601]
Hi
As the IT of my company, I don't know for what reason, in two months, two people called me because it clicked select all at inbox and deleted "accidentally"..
It can be possible to disable that button?
![image](https://user-images.githubusercontent.com/5132053/93067815-f9c3f200-f67b-11ea-9790-2ebe1781202d.png)

**Comment by trasherdk** [comment]
Seems like a server problem. Using delayed delete/expunge settings might help.  

https://www.cyrusimap.org/imap/reference/faqs/o-deleted-expired-expunged-purged.html  
https://www.cyrusimap.org/2.5/imap/features/delayed-delete.html

**Comment by kprkpr** [comment]
Yeah.. We can use that. No problem because backups, I recover all with no problem, the problem is that "users" (At least here), mistakes "Select all" with "Only this page" and deletes all, is for that thing that I ask to hide this icon, maybe css, or something..
